pkgs/applications/editors/vim/plugins/nvim-treesitter/overrides.nix

```diff
@@ -3,11 +3,26 @@
 self: super:
 
 let
-  builtGrammars = callPackage ./generated.nix {
+  generatedGrammars = callPackage ./generated.nix {
     buildGrammar = callPackage ../../../../../development/tools/parsing/tree-sitter/grammar.nix { };
   };
 
-  allGrammars = lib.filter lib.isDerivation (lib.attrValues builtGrammars);
+  generatedDerivations = lib.filterAttrs (_: lib.isDerivation) generatedGrammars;
+
+  builtGrammars = generatedGrammars // lib.listToAttrs
+    (lib.concatLists (lib.mapAttrsToList
+      (k: v:
+        let
+          replaced = lib.replaceStrings [ "_" ] [ "-" ] k;
+        in
+        map (lib.flip lib.nameValuePair v)
+          ([ ("tree-sitter-${k}") ] ++ lib.optionals (k != replaced) [
+            replaced
+            "tree-sitter-${replaced}"
+          ]))
+      generatedDerivations));
```

Comment:
This is generating aliases with - right? Maybe add a really short comment.

Reply:
yes, for example, ocaml_interface will ads ocaml-interface, tree-sitter-ocaml-interface and tree-sitter-ocaml_interface